### PR TITLE
Updates to cc65-sprite and cc65-audio for r37 and latest CC65

### PR DESCRIPTION
**Pull request by CJLove**
With these changes built using the latest CC65 the sprite and audio demos work with Emulator/rom version r37.

### CODE REVIEW

File: cc65-audio/main.c [hunk 1]
```diff
@@ -43,12 +43,12 @@ static uint16_t ofs, start;
 
 static void startWrite()
 {
-    vpoke(0, 0x0f100c);
+    vpoke(0, 0x01f1a0c);
```
Comment (greg-king5) [inline cc65-audio/main.c:46]: That address has too many digits.

Comment (greg-king5) [reply cc65-audio/main.c]: The address must be the same as the address in `endwrite()`.

**Comment by greg-king5** [comment]
:+1:

**Comment by mist64** [comment]
Sorry for the delay!

**Pull request merged**